BP/common/culture/cultures/00_central_germanic.txt

```diff
@@ -0,0 +1,1772 @@
+			Engelbrecht Erich_Erik Ernst_Ernest Ezzo Folkhard Folkmar Franz_Francis Friedrich_Frederick Gebhard Georg_George
+			Gerhard Gerlach Giselbert_Gilbert Gottfried_Godfrey Gotthard Gottschalk Gotzelo Guntram Günzel Hartmann Hartwig 
+			Heinrich_Henry Herbert Hermann_Herman Hesso Hugo_Hugh Humbert_Humbert Jakob_Jacob Johann_John Karl_Charles Karloman
+			Konrad_Konrad Kuno_Konrad Lambert_Lambert Leopold Lothar_Lothaire Ludolf Ludwig_Louis Luitpold Lutbert Magnus_Magnus
+			Manfred Markward Martin_Martin Mathias_Matthew Meinhard Michael_Michael Nikolaus_Nicholas Norbert Ordulf Otto_Otto
+			Peter_Peter Philipp_Philip Poppo Rudger_Roger Reinhard Rudolf Ruprecht_Robert Siegfried Sieghard Siegmund_Sigmund
+			Simon_Simon Stefan_Stephen Theodorich_Theodore Thietmar Thomas_Thomas Udo Ulrich_Ulrich Viktor Waldemar_Vladimir Walram
+			Walther_Walter Welf Wenzel_Vaclav Werner Widukind Wigerich Wilhelm_William Wolfgang Wolfram
+		}
+		female_names = {
+			Adela_Adela Adelheid_Adelaide Adelinde Agathe_Agatha Agnes_Agnes Alberade Amalberga Amalie_Amelia 
+			Barbara Beatrix_Beatrice Benedicta_Benedicta Berchte Berta Binhilde Brunhilde_Brunhilda Cecilie_Cecilia Cothilda_Clotilde Christine_Christina
+			Dorothea_Dorothy Elisabeth_Elisabeth Emma_Emma Engeltraud Ermengard_Ermengarde Eufemia Friedrica_Frederique Frida 
+			Gerberga_Gerberga Gerhild Gertrude_Gertrude Gisela_Gisela Gudrun_Gudrun Gunhilda_Gunhild Hedwig Heilwiva Heike Helene_Helen 
+			Helga Henrike Hildeburg Hildegard_Hildegard Ida_Ida Ilsa Inge_Inge Irmele Irmeltrud Irmgard_Ermengarde Isentrude
+			Judith_Judith Jutte Katharina_Catherine Karlotte Klara Klementia_Clementia Konstanze_Constance Kriemhild Kunigunde 
+			Luitgard Margarete_Margaret Maria_Maria Mathilde_Matilda Mechtild Oda Osterhild Richara_Ricarda Richenza Richwara Romilda 
+			Serhilda Sofie_Sophia Sybille_Sibylla Trude Ulrike Ursula Ute Walpurga Wilhelmina Winfrida Wulfhilde
+		}
+		dynasty_of_location_prefix = "dynnp_von"
+		patronym_suffix_male = "dynnpat_suf_son"
+		patronym_suffix_female = "dynnpat_suf_son"
+
+		# Chance of male children being named after their paternal or maternal grandfather, or their father. Sum must not exceed 100.
+		pat_grf_name_chance = 50
+		mat_grf_name_chance = 5
+		father_name_chance = 10
+		
+		# Chance of female children being named after their paternal or maternal grandmother, or their mother. Sum must not exceed 100.
+		pat_grm_name_chance = 10
+		mat_grm_name_chance = 50
+		mother_name_chance = 5
+
+		ethnicities = {
+			25 = caucasian_blond
+			15 = caucasian_ginger
+			35 = caucasian_brown_hair
+			25 = caucasian_dark_hair
+		}
+
+		mercenary_names = {
+			{ name = "mercenary_company_great_company" coat_of_arms = "mc_great_company" }
+			{ name = "mercenary_company_veterans_of_the_marches" }
+			{ name = "mercenary_company_swiss_guard" }
+		}
+	}
+
+	yiddish = { # Unused, language of Ashkenazi
+		
+		color = { 0.49 0.41 0.24 }
+
```

Comment:
You should also port the dynasty names.

Reply:
Will do that too. I thought we'd be leaving porting for later so that's why I left it.

Reply:
Oh wait I messed up what you mean. Yeah I'll do that now

Reply:
I ported that with the rest, but Yiddish in particular was unused so it is not really necessary to keep it. We can remove it entirely.

Reply:
Oh you've already ported them? Also yeah okay then I'll remove it

Reply:
I ported them for the cultures I did, yes

Reply:
Yeah, that's what I was asking. Ported them from where. I'm thinking of like going into the history files and maybe just like finding some of the dynasties of the involved cultures and just using their names I guess.

Reply:
From _BP\common\dynasties\BP_dynasties.txt_

Reply:
Huh thanks. Lol I edited that file here and forgot

Reply:
Actually also should I leave out "low_german"? Same deal with Yiddish in that it's unused.

Reply:
You can leave it out as well

Reply:
Okay, all is done, though "bucinobantian" and "berber_germanic" have no dynasties.

Reply:
`berber_germanic ` can be removed too. Does `founder_named_dynasty = yes` work in CK3?

Reply:
Yeah it does

Reply:
Ok, but it is not used by vanilla, hence my confusion

Reply:
It is in Vanilla. The best example is the Arabic cultures.

Reply:
My bad in that case, I confused it with another removal-